doc/LSTM_MODEL.md

```diff
@@ -0,0 +1,48 @@
+# LSTM Model
+
+* [Summary](#summary)
+* [Formulation Config](#formulation-config)
+    * [Required Parameters](#required-parameters)
+
+## Summary
+
+A long short-term memory (LSTM) model and corresponding realization is included in the ngen framework. The model is not included in the default ngen build because it requires the LibTorch library in order to compile and run, and this is not a basic required library for ngen. 
+
+The basic outline of steps needed to run the LSTM model is:
+  * [Install the LibTorch library version 1.8 or later.](https://pytorch.org/docs/stable/cpp_index.html)
+  * Create the build directory including the option to activate the lstm: 
+  
+      `cmake -DCMAKE_BUILD_TYPE=Debug -B cmake-build-debug -DLSTM_TORCH_LIB_ACTIVE:BOOL=ON -S .`  
+  
+  * Unit tests for the LSTM model and realization can then be built and run from the main directory with the following two commands:
+  
+      `cmake --build cmake-build-debug --target test_lstm`  <br />
+      
+      `./cmake-build-debug/test/test_lstm`  
+  
+  * The formulation config and required parameters to run the LSTM for a given catchment are below.
+
+## Formulation Config
+An example realization is ngen/data/lstm/example_lstm_realization_config.json.
```

Comment:
make this a link` [example_lstm_realization_config.json](ngen/data/lstm/example_lstm_realization_config.json)`